tests/common/test_cgroups.py

```diff
@@ -62,7 +63,10 @@ def path_maker(hierarchy, _):
 def i_am_root():
     return os.geteuid() == 0
 
+def is_wsl():
+    return '-Microsoft-' in platform.platform()
 
+@skip_if_predicate_true(is_wsl, "CGroups not supported under WSL")
```

Comment:
All this does is suppress the test; it doesn't keep the agent code itself from trying to do the CGroup thing. The better fix would be to arrange for CGroups.enabled to always be False if it's running under WSL.

Is there an osutil subclass for WSL? If we created one, that would be the place to insert the necessary code to cause CGroups functionality to be completely disabled (including the tests).

Reply:
In osutil/ubuntu.py:
```
    def is_cgroups_supported(self):
        if ('TRAVIS' in os.environ and os.environ['TRAVIS'] == 'true') \
                or ('-Microsoft-' in platform.platform()):
            return False
        return True
```

Reply:
Ok, updated